src/InContextSignup.js

```diff
@@ -33,8 +33,9 @@ export class InContextSignup {
 
     isAvailable () {
         const isEnabled = this.device.settings?.featureToggles.emailProtection_incontext_signup
+        const isLoggedIn = this.device.hasLocalAddresses
```

Comment:
Can we rather use `isDeviceSignedIn`? I'm afraid we may not have local addresses on all platforms, but all platforms will return true to `isDeviceSignedIn`.